bukkit/src/main/java/me/lucko/luckperms/bukkit/LPBukkitBootstrap.java

```diff
@@ -311,4 +313,13 @@ private static boolean checkIncompatibleVersion() {
             return true;
         }
     }
+
+    private static boolean isFolia() {
+        try {
+            Class.forName("io.papermc.paper.threadedregions.scheduler.RegionScheduler");
```

Comment:
```suggestion
            Class.forName("io.papermc.paper.threadedregions.RegionizedServer");
```

paper api now includes the folia scheduler stuff